docs/content/en/docs/concepts/tracing-policy/hooks.md

```diff
@@ -400,6 +400,61 @@ The `maxData` flag does not work with `returnCopy` flag at the moment, so it's
 usable only for syscalls/functions that do not require return probe to read the
 data.
 
+### Advanced usage
+
+For specific use cases, you may want to extract a specific attribute from the argument.
+For instance you have `struct linux_binprm` as first argument and want to filter parent
+process name, you can do it as following.
+
+```yaml
+apiVersion: cilium.io/v1alpha1
+kind: TracingPolicy
+metadata:
+  name: "lsm"
+spec:
+  lsmhooks:
+  - hook: "bprm_check_security"
+    args:
+      - index: 0
+        type: "linux_binprm"
+        extractParam: "mm.owner.real_parent.comm"
+        overwriteType: "string"
+    selectors:
+      - matchActions:
+        - action: Post
+```
+
+The above policy will display the parent process name every time the hook is called.
+The `extractParam` field is used to reach a specific data into the `struct
+linux_binprm`. It is important to set `overwriteType` as well to make sure the
+reached data is read correctly (as a string in this case).
```

Comment:
```suggestion
The above policy will display the parent process's comm every time the hook is called.
The `extractParam` field is used to reach specific data into the `struct linux_binprm`. 
It is important to set `overwriteType` as well to make sure the reached data is read 
correctly (as a string in this case).
```

I would say `comm` maybe instead of name since this value can be changed by the thread itself, it's not a really safe way of identifying a process name.